Makefile

```diff
@@ -159,25 +159,29 @@ $(SCOPE_UI_TOOLCHAIN_UPTODATE): client/yarn.lock $(SCOPE_UI_BUILD_UPTODATE)
 	touch $(SCOPE_UI_TOOLCHAIN_UPTODATE)
 
 client/build/index.html: $(shell find client/app -type f) $(SCOPE_UI_TOOLCHAIN_UPTODATE)
-	mkdir -p client/build
 	if test "true" != "$(SCOPE_SKIP_UI_ASSETS)"; then \
+		rm -rf client/build; \
```

Comment:
This seems to be deleting the cache on every build?

Reply:
Yes @bboreham .

Reply:
@bboreham That's actually what `CleanWebpackPlugin` is already doing in our current setup and I imagine that's exactly the first point of failure which makes #3612 occur in the first place.